RTN-004.tex

```diff
@@ -282,35 +305,47 @@ \subsection{Application Form Contents}\label{ssec:sel_form}
 \item time commitment (yes/no to confirm recognition of minimum expected time commitment)
 \item delegate responsibilities (yes/no to confirm recognition of expectations and responsibilities)
 \item data rights (yes/no/idk to confirm data rights)
+\item origin of data rights for applicants outside the US and Chile (drop-down menu if possible)
+\item in-kind proposal team association (drop-down if possible, write-in if not)
 \end{itemize}
 
+Regarding the applicant's confirmation of data rights, the form should make clear that all astronomers working in the US and Chile have data rights (including students), and that those individuals should select "yes".
+
+Applicants from outside the US and Chile will indicate the origin of their data rights: either via an agreement between their local institution and the NSF, DOE, AURA, SLAC or LSST Project, or via an in-kind proposal to the US agencies. 
+For the former, the selection committee will be able to verify the data rights based on the applicant's name and institution.
+For the latter, the selection committee can verify the applicant's data rights status by contacting the relevant in-kind proposals program manager.
+Applicants associated with in-kind proposals that are still in the approval process at the time of their DP0 application should be directed to choose "I don't know" and provide the in-kind proposal team association information. 
 
 \subsection{Selection Timeline}\label{ssec:sel_time}
 
+Applications will be primarily reviewed by the selection committee in two stages, which coincide with the projected dates when the DP0.1 and DP0.2 science-ready data products will be made available to DP0 delegates in the RSP (Section \ref{ssec:intro_stages}).
```

Comment:
You avoided mention of "science-ready" -- do you want to keep this terminology here?

Reply:
No! Thank you! I did a search-and-replace on "science-ready" to be safe. No more appearances.